src/components/game/game.actions.ts

```diff
@@ -1,14 +1,14 @@
 import { createAction, ActionsUnion } from 'src/utils/actionHelpers'
-import { TileMatrix } from './grid.helpers'
+import { TileMatrix } from './game.helpers'
 
 export enum ActionTypes {
-    INIT_GRID = '[grid] init grid',
-    INIT_GRID_SUCCESS = '[grid] init grid success'
+    INIT_GAME = '[game] init game',
+    INIT_GRID = '[game] init grid'
 }
 
 export const actions = {
-    initGrid: () => createAction(ActionTypes.INIT_GRID),
-    initGridSuccess: (tiles: TileMatrix) => createAction(ActionTypes.INIT_GRID_SUCCESS, tiles)
+    initGame: () => createAction(ActionTypes.INIT_GAME),
+    initGridSuccess: (tiles: TileMatrix) => createAction(ActionTypes.INIT_GRID, tiles)
```

Comment:
this doesn't look right:
initGridSuccess => ActionTypes.INIT_GRID

Reply:
This was fixed in another PR :)